spec/models/ext_management_system_spec.rb

```diff
@@ -396,6 +396,91 @@
     end
   end
 
+  context "#pause!" do
+    it 'disables an ems with child managers and moves parent to maintenance zone' do
+      zone  = FactoryGirl.create(:zone)
+      ems   = FactoryGirl.create(:ext_management_system, :zone => zone)
+      child = FactoryGirl.create(:ext_management_system, :zone => zone)
+      ems.child_managers << child
+
+      ems.pause!
+
+      expect(ems.enabled).to be_falsy
```

Comment:
`expect(ems.enabled?).to eq(false)` We should expect false, not anything non-true (nil).

Reply:
Can we do `expect(ems).to be_enabled`?